custom_components/cololight/light.py

```diff
@@ -196,8 +196,7 @@ def is_on(self):
 
     @property
     def should_poll(self) -> bool:
-        """Turn off, as cololight doesn't share state info"""
-        return False
+        return True
```

Comment:
Looks like the default scan interval is 30s.
Think this is reasonable for this device, or should we increase it?

Reply:
Having a look what other components have done, feeling 30s poll is reasonable.